core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -502,7 +502,7 @@ public Address getAddress() {
       CompletableFuture<Map<Address, Response>> future = invokeRemotelyAsync(recipients, rpcCommand, mode,
             timeout, responseFilter, deliverOrder, anycast);
       try {
-         return future.get();
+         return CompletableFutures.await(future);
```

Comment:
@pruivo Please add a comment saying the `invokeRemotelyAsync` is guaranteed to complete within `timeout` millis, so there's no need to use a timeout here.